src/WeatherOnMarsImage/WeatherOnMarsImage.tsx

```diff
@@ -0,0 +1,18 @@
+﻿import React from "react";
+import './WeatherOnMarsImage.scss';
+
+function WeatherOnMarsImage() {
+
+
+    return (
+        <div className='iframeContainer'>
+            <iframe
+                src='https://mars.nasa.gov/layout/embed/image/insightweather/'
+                height ='530'
+                width = '750'
+            />
```

Comment:
I'm guessing this isn't as responsive as we'd ideally like?

Reply:
It is contained within a responsive div so is relatively responsive -
I'm not sure if those height and width values do anything any more. I might remove them..

laptop:
![1024 laptop](https://user-images.githubusercontent.com/60605747/75879477-64fe5300-5e13-11ea-9be2-6da61734af24.PNG)

iphone X:
![iphone X](https://user-images.githubusercontent.com/60605747/75879481-662f8000-5e13-11ea-92ee-a4cda7b47aa0.PNG)

Reply:
I have removed them now